src/main/java/blockchains/iaas/uni/stuttgart/de/plugin/bitcoin/BitcoinConnectionProfile.java

```diff
@@ -0,0 +1,142 @@
+package blockchains.iaas.uni.stuttgart.de.plugin.bitcoin; /*******************************************************************************
+ * Copyright (c) 2019 Institute for the Architecture of Application System - University of Stuttgart
```

Comment:
If you introduced any sort of changes to this file, please updated the year and add yourself as a co-author

Reply:
I did not change this class. But, I will add myself as co-author in other classes.